ServerRunner/ServerRunner.cs

```diff
@@ -10,31 +10,27 @@ public class ServerRunner
     {
         public static void Main()
         {
-            Console.WriteLine("Path to DLL containing the distributed task:");
-            var distributedTaskAssemblyPath = Console.ReadLine();
-            var distributedTaskAssembly = Assembly.LoadFrom(distributedTaskAssemblyPath);
-            var distributedTask = GetTypeFromAssembly<IDistributedTask>(distributedTaskAssembly);
-
             /**
-             * The DLL with the ITask is loaded here just in order to run the task and for the IDistributedTask
+             * The DLL with the ISubtask and ITask is loaded here just in order to run the task and for the ITask
              * to be able to aggregate the results.
              * 
              * In the final version the input data will be sent to the distributed nodes who will be responsible
-             * for executing the ITask compiled to JS.
+             * for executing the ISubtask compiled to JS.
              */
             Console.WriteLine("Path to DLL containing the task:");
             var taskAssemblyPath = Console.ReadLine();
             var taskAssembly = Assembly.LoadFrom(taskAssemblyPath);
             var task = GetTypeFromAssembly<ITask>(taskAssembly);
+            var distributedTask = GetTypeFromAssembly<ISubtask>(taskAssembly);
```

Comment:
Lets' change the name of this variable to `subtask` to avoid ambiguities :smile: